x/staking/client/cli/query.go

```diff
@@ -116,7 +116,7 @@ $ %s query staking validators
 				return err
 			}
 
-			var validators types.Validators
+			var validators []types.Validator
```

Comment:
why the change?

Reply:
reverting, I had tried migrating to proto in this PR but instead pushed it to #6987, this was leftover